parser/tools/postbuild-wasm.mjs

```diff
@@ -213,7 +213,7 @@ async function generateEmbedded(code, profile) {
 
       const replacement = parseExpression(`
         function loadWASM() {
-          return Buffer.from('${payload}', 'base64');
+          return Uint8Array.from(globalThis.atob('${payload}'), c => c.codePointAt(0))
```

Comment:
This is probably not needed. This file is only used when building Milo, so not at runtime (either on Node or on the browser).

Reply:
My understanding is that this is what generates the index-with-wasm file. This replaces the node require that you commented on.

Reply:
Yes, you are right, I misread the snippet. Sorry about it.